eng/dependabot/Packages.props

```diff
@@ -0,0 +1,32 @@
+<Project>
+  <!-- Packages in this file have versions updated periodically by Dependabot. Versions managed by Darc/Maestro should be in ..\Package.props. -->
+  <ItemGroup>
+    <PackageReference Update="System.Diagnostics.Process" Version="4.3.0" />
+    <PackageReference Update="System.IO.Compression" Version="4.3.0" />
+    <PackageReference Update="System.Runtime.Loader" Version="4.3.0" />
+    <PackageReference Update="Microsoft.Build.Framework" Version="16.10.0" />
+    <PackageReference Update="Microsoft.Build.Utilities.Core" Version="16.10.0" />
+    <PackageReference Update="Newtonsoft.Json" Version="13.0.1" />
+    <PackageReference Update="Wcwidth.Sources" Version="0.5.0" />
+    <PackageReference Update="Microsoft.Extensions.Logging" Version="7.0.0-preview.5.22272.3" />
+    <PackageReference Update="Microsoft.Extensions.Logging.Console" Version="7.0.0-preview.5.22272.3" />
+    <PackageReference Update="Microsoft.Extensions.Logging.Abstractions" Version="7.0.0-preview.5.22272.3" />
+    <PackageReference Update="NuGet.Configuration" Version="6.2.0-rc.146" />
+    <PackageReference Update="NuGet.Credentials" Version="6.2.0-rc.146" />
+    <PackageReference Update="NuGet.Protocol" Version="6.2.0-rc.146" />
+    <!--Analyzers-->
+    <PackageReference Update="Microsoft.Net.Compilers.Toolset" Version="4.2.0-4.22266.5" />
+    <PackageReference Update="Microsoft.CodeAnalysis.CSharp.CodeStyle" Version="4.2.0-4.22266.5" />
+    <PackageReference Update="Microsoft.CodeAnalysis.NetAnalyzers" Version="7.0.0-preview1.22277.2" />
+    <PackageReference Update="Microsoft.CodeAnalysis.PublicApiAnalyzers" Version="3.3.4-beta1.22277.2" />
+    <PackageReference Update="StyleCop.Analyzers" Version="1.2.0-beta.333" />
+    <!--Test dependencies-->
+    <PackageReference Update="FakeItEasy" Version="6.2.1" />
+    <PackageReference Update="FluentAssertions" Version="5.10.3" />
+    <PackageReference Update="Microsoft.DotNet.Cli.Utils" Version="6.0.100-rc.1.21423.4" />
```

Comment:
`Microsoft.DotNet.Cli.Utils` should be managed via maestro. It is not a public package.

Reply:
Where can I get the information similar to below for `Microsoft.DotNet.Cli.Utils` in Version.Details.xml? From Nuget Package Manager in VS I know its Uri is https://github.com/dotnet/sdk. What is the Sha for specific version?
```
    <Dependency Name="Microsoft.Extensions.Logging" Version="7.0.0-preview.6.22279.1">
      <Uri>https://github.com/dotnet/runtime</Uri>
      <Sha>1466e404dfac7ad6af7e6877d26885ce42414120</Sha>
    </Dependency>
```

Reply:
It's in sdk repo. Let's keep it on static version.  Likely we will no longer need it after repo merge.